src/Numerics/ODESolvers/MultirateInfinitesimalStepMethod.jl

```diff
@@ -151,6 +198,49 @@ mutable struct MultirateInfinitesimalStep{
     end
 end
 
+function MultirateInfinitesimalStep(
```

Comment:
@vogel-d Is this used? I don't see an instance of this being used anywhere

Reply:
It is used when MIS is called as a fast solver, e.g. in the rising bubble test (l. 112-119):
`fast_method = (dg, Q, nsubsteps) ->
                    MultirateInfinitesimalStep(
                        MISKWRK43,
                        dg,
                        (dgi, Qi) -> LSRK54CarpenterKennedy(dgi, Qi),
                        Q,
                        nsubsteps = nsubsteps,
                    ),`

This could probably alternatively be achieved by
`fast_method = (dg, Q, nsubsteps) ->
                    MISKWRK43(
                        dg.rhs![1],
                        dg.rhs![2],
                        (dgi, Qi) -> LSRK54CarpenterKennedy(dgi, Qi),
                        nsubsteps,
                        Q
                    ),`
and then the outer constructor could be avoided. 

I defined similar outer constructors for MRK and ARK as fast solvers to make it cleaner and to avoid confusion regarding the data type of dg and TimeScaledRHS.